presto-raptor/src/test/java/com/facebook/presto/raptor/storage/TestOrcStorageManager.java

```diff
@@ -219,6 +229,36 @@ public void testWriter()
             assertEquals(stringVector.vector[0], utf8Slice("hello"));
             assertEquals(stringVector.vector[1], utf8Slice("bye"));
 
+            SingleObjectVector arrayVector = new SingleObjectVector();
```

Comment:
I can remove them if you want, but these seem to test it better than `TestShardWriter` as they uncovered bugs even though the shard writer tests passed